packages/pds/src/db/index.ts

```diff
@@ -193,8 +193,17 @@ export class Database {
     return txRes
   }
 
-  async txAdvisoryLock(name: string): Promise<boolean> {
+  async takeTxAdvisoryLock(name: string): Promise<boolean> {
     this.assertTransaction()
+    return this.txAdvisorylock(name)
+  }
+
+  async checkTxAdvisoryLock(name: string): Promise<boolean> {
+    this.assertNotTransaction()
+    return this.txAdvisorylock(name)
```

Comment:
Clever/interesting—I assume it takes and releases the lock during the course of the statement?  I wonder if there's a simple way to check without taking it at all. 🤔 May not really matter!